src/web/src/hooks/use-telescope-contributor.ts

```diff
@@ -12,31 +11,80 @@ export type Contributor = {
 const telescopeGitHubContributorsUrl =
   'https://api.github.com/repos/Seneca-CDOT/telescope/contributors';
 
-const randomContributor = (contributors?: any[]) =>
-  contributors ? contributors[Math.floor(Math.random() * contributors.length)] : null;
+const prepareURL = (page?: number) =>
+  `${telescopeGitHubContributorsUrl}?per_page=1&${page ? `page=${page}` : ''}`;
 
-const PER_PAGE: number = 50;
+const linkParser = (links: string, key: string) => {
```

Comment:
I'm pretty sure we have this code in the frontend somewhere else already, I think @Grommers00 or @manekenpix wrote it for paging.  Can we reuse what we have?

Reply:
@humphd I made some changes to make this more hooks-based, but I couldn't find in the frontend where a similar regex was written.